src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: #fff;
+  margin: 0;
+}
+
+.logo-link {
+  margin: 0;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: inline-block;
+  justify-content: space-between;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-item {
+  display: flex;
+  /* height: 60px; */
+  align-items: center;
+  margin: 0 10px;
```

Comment:
you need only one side margin (either margin-right or margin-left). 
Use either `:not(:last-child)` or `:not(:first-child)`